adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/db/InMemoryExecParamRepository.java

```diff
@@ -137,10 +137,10 @@ public ExecParamParser getParser(final String parameterJsonString,
                                      final String[] args,
                                      final Logger logger) {
         if (args.length != 0) {
-            logger.info("Retrieving execution parameters from command-line");
+            logger.info("Retrieving execution parameters from command-line\n");
```

Comment:
To make this platform-independent, you should use `System.lineSeparator()` instead of `\n`.

See more here - https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--

Reply:
I had no idea about that, thks! Please disregard my approval and implement the change requested by Sean.